cloudfoundry/azure-cloud-foundry-service-sample/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>1.5.4.RELEASE</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+
+	<groupId>azure.cloud.foundry.service.sample</groupId>
+	<artifactId>azure-cloud-foundry-service-sample</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>Azure Cloud Foundry Service Sample</name>
+	<description>Demo project for Spring Boot</description>
+	<url>https://github.com/Microsoft/azure-spring-boot-starters</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<java.version>1.8</java.version>
+	</properties>
+
+	<licenses>
+		<license>
+			<name>MIT</name>
+			<url>https://github.com/Microsoft/azure-spring-boot-starters/blob/master/LICENSE</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<id>ZhijunZhao</id>
+			<name>Zhijun Zhao</name>
+			<email>zhijzhao@microsoft.com</email>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>scm:git:git://github.com/Microsoft/azure-spring-boot-starters.git</connection>
+		<developerConnection>scm:git:ssh://github.com:Microsoft/azure-spring-boot-starters.git</developerConnection>
+		<url>https://github.com/Microsoft/azure-spring-boot-starters/tree/master</url>
+	</scm>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.microsoft.azure</groupId>
+			<artifactId>azure-storage-spring-boot-starter</artifactId>
+			<version>0.1.4</version>
```

Comment:
If you don't want this sample to reference `azure-spring-boot-starter-bom` as  parent, can you use dependency management here to import the bom module? So, we don't need to specify 0.1.4 for each starter.